lib/response.js

```diff
@@ -158,6 +160,11 @@ res.send = function send(body) {
         if (!this.get('Content-Type')) {
           this.type('bin');
         }
+      } else if (isChunkBlob) {
+        if (!this.get('Content-Type')) {
+          this.type(chunk.type || 'bin');
```

Comment:
bin?

🤔 🧐 🤨

Are we sure that we want to default to some `content-type` and if anything shouldn't it be `application/octet-stream`?

Reply:
Yea, that is what `bin` is. I think there is a test in the pr that confirms this already.

Reply:
yes, it is already there